### PR TITLE
Disables nick options dropup in chat input box when no room is selected.

### PR DESCRIPTION
**Pull request by mjnovice**
Fixes issue #49.

### CODE REVIEW

File: waartaa/client/chat/chat_input.js [hunk 1]
```diff
@@ -5,6 +5,11 @@ Template.chat_input.events({
     if (keyCode == 9)
       e.preventDefault();
   },
+  'click .input-group-btn':function(e) {
```
Comment (rtnpro) [inline waartaa/client/chat/chat_input.js]: Not an elegant solution :-1:

**Comment by mjnovice** [comment]
rtnpro, could you suggest something better ?

**Comment by rtnpro** [comment]
@mjnovice Please rebase your branch on top of upstream's master and update this PR.

```
git pull --rebase upstream master
git push origin master
```

where `upstream` is pointing to `https://github.com/waartaa/waartaa` and `origin` is pointing to your fork.

**Comment by rtnpro** [comment]
Accepted your commit manually. However, I needed to do some cleanup here fca7f5fff98aba5078774273c8802db27c934f53. Please stick to standard coding guidelines when submitting patches upstream. Also, never have non linear history in your PR.

Never merge, always rebase,

**Pull request closed without merging**